README.md

```diff
@@ -1,2 +1,120 @@
+
+| Parameter | Type | Description | Required | Default |
+| --- | --- | --- | --- | --- |
+| `version-path` | string | File path to file containing the version. If not specified will default to version.py. We will split the line matching `version=x.y.z` to get the version. | false |  `version.py`|
+| `changelog-path` | string | File path to the changelog containing the description of the release. The format should follow typical [Changlog Symantics](https://keepachangelog.com/en/1.0.0/). | false | `CHANGELOG.md` |
+| `file-path` | string | File directory to be uploaded in addition to the full repository. | false | `""`|
+| `is-draft` | boolean | If true, then the release is a draft. | false | `false` |
+| `is-pre-release` | boolean | If true, then the release is a pre-release. | false |`false` |
+
+### Dependencies / Requirements
+We have checks in place to make sure the following are installed within the container otherwise an error message will be logged and task failed. If your job uses the `github` executor, these dependencies will be installed. Otherwise, we recommend the following Docker images that will ensure these are downloaded:
+
+* cibuilds/github:0.13
+
+
+#### Bash Shell
+These commands require `Bash` in order to function. `Bash` is the default shell for CircleCI; therfore, this Orb should be compatible with most images.
+
+#### cURL
+cURL is used to get TAG data for the GitHub releases and must be installed in the container to function.
+
+#### ghr
+The tool that does the creation of the GitHub Release.
+
+#### awk
+Only the `release-by-version` is dependent upon `awk` to function and it is already installed with the executor for this job. We use this in order to parse the version and changelog files.
+
+## Development
+
+#### Prerequisites
+Make sure you have CircleCI CLI installed on your machine. If you don't, you can follow [Setting Up the CircleCI CLI](https://circleci.com/docs/2.0/orb-author-cli/#section=configuration) to get setup.
+
+### Building and Publishing the Orb
+Once ready to publish a new version of the orb you can follow these steps:
+
+1. Build the orb by running: `circleci config pack src > orb.yml`. Do not commit the created `orb.yml` file. This file is for publishing to CircleCI when finished.
+2. Once the orb is built, we can verify the orb is valid by running `circleci orb validate orb.yml`.
+3. Publish a dev version of the orb by running `circleci orb publish orb.yml civisanalytics/github-release@dev:<version>`. This dev version is available only to civisanalytics and will expire 90 days after each publish. This can be used for testing the orb before a final release.
+4. When ready to publish the orb, email IT at it-help@civisanalytics.com to have them run `circleci orb publish promote civisanalytics/github-release@dev:<version> <segment>` where `segment` is either `major`, `minor`, or `patch` and it will auto-increment the published version of the orb.
```

Comment:
If this is going to be public, do we want to put our internal IT helpdesk email in the Readme?

Reply:
hmm should it be something like `when ready to publish the orb, contact a Civis employee`

Reply:
also only IT members can publish an orb not all Civis employees

Reply:
I'm not sure I like that phrasing either. I think I want to just say something like "the orb owner can run `circleci orb publish promote civisanalytics/github-release@dev:<version> <segment>`. That way, we don't mention us/IT at all.